### ISSUE TITLE
Multi-level sampling

### ISSUE THREAD

**Issue by yannikschaelte** [ICB-DCM/pyABC#172]
Allow to use surrogate models in non-final iterations, and only the full, expensive, model in the last iteration. This will make things more efficient. What to do? One way that would suffice would be to make the model simulation time-aware (pass t to .sample). There might also be some more handy alternatives.

**Comment by ljschumacher** [comment]
This reference may be of interest: 
"Multifidelity Approximate Bayesian Computation with Sequential Monte Carlo Parameter Sampling"
https://arxiv.org/abs/2001.06256

**Comment by yannikschaelte** [comment]
Hi @ljschumacher . Thanks, this is exactly one of the ways we had in mind :smile:. Are you interested in using such a method?

**Comment by ljschumacher** [comment]
Yes, I would be. We have a use case where a faster deterministic model may help speed up inference for the full stochastic one
